diff --git a/include/linux/tty.h b/include/linux/tty.h
index 8c76fd97d..61908168c 100644
--- a/include/linux/tty.h
+++ b/include/linux/tty.h
@@ -65,7 +65,7 @@
 #define I_IGNCR(tty)	_I_FLAG((tty), IGNCR)
 #define I_ICRNL(tty)	_I_FLAG((tty), ICRNL)
 #define I_IUCLC(tty)	_I_FLAG((tty), IUCLC)
-#define I_IXON(tty)	_I_FLAG((tty), IXON)
+#define I_IXON(tty)	0
 #define I_IXANY(tty)	_I_FLAG((tty), IXANY)
 #define I_IXOFF(tty)	_I_FLAG((tty), IXOFF)
 #define I_IMAXBEL(tty)	_I_FLAG((tty), IMAXBEL)
