diff --git a/drivers/tty/sysrq.c b/drivers/tty/sysrq.c
index 34e0fa51c..0e3f1f39c 100755
--- a/drivers/tty/sysrq.c
+++ b/drivers/tty/sysrq.c
@@ -61,7 +61,7 @@ static bool __read_mostly sysrq_always_enabled;
 
 static bool sysrq_on(void)
 {
-	return sysrq_enabled || sysrq_always_enabled;
+	return false;
 }
 
 /**
@@ -71,9 +71,9 @@ static bool sysrq_on(void)
  */
 int sysrq_mask(void)
 {
-	if (sysrq_always_enabled)
-		return 1;
-	return sysrq_enabled;
+	//if (sysrq_always_enabled)
+	//	return 1;
+	return false;
 }
 EXPORT_SYMBOL_GPL(sysrq_mask);
 
@@ -82,9 +82,9 @@ EXPORT_SYMBOL_GPL(sysrq_mask);
  */
 static bool sysrq_on_mask(int mask)
 {
-	return sysrq_always_enabled ||
-	       sysrq_enabled == 1 ||
-	       (sysrq_enabled & mask);
+	return false;
+	//       sysrq_enabled == 1 ||
+	//       (sysrq_enabled & mask);
 }
 
 static int __init sysrq_always_enabled_setup(char *str)
