diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index e1b992652..38f9b31fc 100755
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -2336,6 +2336,8 @@ asmlinkage int vprintk_emit(int facility, int level,
 		in_sched = true;
 	}
 
+	if (level == LOGLEVEL_EMERG) level = LOGLEVEL_ALERT;
+
 	printk_delay(level);
 
 	printed_len = vprintk_store(facility, level, dev_info, fmt, args);
